kinde_sdk/frontend/api_client.py

```
@@ -0,0 +1,798 @@
+from urllib.parse import quote
+from typing import Tuple, Optional, List, Dict, Union
+from pydantic import SecretStr
+
+from kinde_sdk.frontend.configuration import Configuration
+from kinde_sdk.frontend.api_response import ApiResponse, T as ApiResponseT
+import kinde_sdk.frontend.models
+from kinde_sdk.frontend import rest
+from kinde_sdk.frontend.exceptions import (
+    ApiValueError,
+    ApiException,
+    BadRequestException,
```

Ruff (0.12.2)

`kinde_sdk.frontend.exceptions.BadRequestException` imported but unused

Remove unused import

(F401)